### ISSUE TITLE
Crash with svg elements

### ISSUE THREAD

**Issue by asabil** [TylorS167/mostly-dom#31]
When updateElement is called on an svg element it crashes when trying to remove the previous classes:

```
Uncaught TypeError: node.className.split is not a function
    at updateElement (updateElement.js:7)
    at patchVNode (patchVNode.js:8)
    at updateChildren (updateChildren.js:25)
    at patchVNodeChildren (patchVNodeChildren.js:11)
    at patchVNode (patchVNode.js:15)
    at patch (init.js:27)
```

**Comment by Frikki** [comment]
Could you give an example code to reproduce the issue? Have you tried adding a `key` on the SVG (I don’t know if that will help)?

**Comment by asabil** [comment]
The problem is quite simple really: `updateElement` assumes that className is a string, which is not true for svg elements, className is `SVGAnimatedString` in the case of svg elements: https://developer.mozilla.org/en-US/docs/Web/API/Element/className#Notes

**Comment by TylorS** [comment]
Thanks for opening the issue. This should be fixed in v4.2.0, I added a test for it!
